data/brands/amenity/fast_food.json

```
@@ -2585,16 +2585,16 @@
       }
     },
     {
-      "displayName": "Hungry Jacks",
+      "displayName": "Hungry Jack's",
       "id": "hungryjacks-d43c69",
       "locationSet": {"include": ["au"]},
```

```suggestion
      "locationSet": {"include": ["au"]},
      "matchNames": ["Hungry Jacks"],
```

You'd better add a matchName for old name